### DISCUSSION TITLE
Easyflash on U64?

### DISCUSSION THREAD

**Discussion by adaumann** [drmortalwombat/oscar64#167] [General]
Some users reported that my EasyFlash game Orbix does not run on Ultima64. I don't know much about it, I thought Easyflash should run on it. How you any idea? I think you have access to my Github, did I made something wrong?

**Comment by drmortalwombat** [comment]
I have no Ultimate64 to test with, but the game works fine with my UltimateII

**Reply by adaumann** [reply]
Someone has fixed it for Ultimate64. Maybe this is interesting for you to review: (german only)

https://www.forum64.de/index.php?thread/151311-new-release-orbix-c64/&postID=2210702#post2210809
